### DISCUSSION TITLE
I'm eager to know what's template.js

### DISCUSSION THREAD

**Discussion by itaim18** [vercel/next.js#54065] [Help]
I am trying to deeply know all of next.js conventions are, template.js looked out to me.
And would like to learn more on all secret features coming ahead

**Comment by icyJoseph** [comment]
- https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#templates from the current docs
- https://nextjs.org/blog/layouts-rfc#templates from the original RFC
